cruds_adminlte/templates/cruds/create.html

```diff
@@ -1,4 +1,4 @@
-{% extends "cruds/base.html" %}
+{% extendstemplate_father %}
```

Comment:
Thank you @matejv, 4 eyes always are better than 2... and we need collaborators here ;)